ui/raidboss/data/02-arr/raid/t10.js

```diff
@@ -111,18 +110,18 @@
       'locale': 'fr',
       'replaceSync': {
         'Imdugud': 'Imdugud',
-        'The Alpha Concourse': 'secteur des croyants',
       },
       'replaceText': {
+        '1x Son': 'Adds 1x Fils',
```

Comment:
This is confusing to read the replacements.  If you're going to replace more than just "son" and "daughter", maybe replacing the whole line would be easier to read, e.g. `'1x Son / 1x Daughter Adds': 'Adds 1x Fils / 1x Fille',`

Reply:
I tried a lot of things, each time either (add) is positioned badly, or (x1, x2). I will try again.

Reply:
- I changed, but I really need the 2 lines with different numbers for it to work.
- Thank you, what I did was complicated for nothing.